woocommerce/woocommerce-ios
# Products Onboarding: Remove unused web auth code

Closes: https://github.com/woocommerce/woocommerce-ios/issues/8097

## Description

This PR removes unused `RequestAuthenticator` and its dependencies from `WebKitViewController`.

## Testing

1. Go to the Products tab and open existing product draft.
2. Tap the Preview button in the navbar.
3. Confirm that you can see a webview with product preview on your store.

---
